README.md

```diff
@@ -117,6 +119,20 @@ The node registers itself with the gateway so the gateway will recognize message
 | app_s_key | string | yes | Application Session Key (16 bytes in hex) Used to decrypt uplink messages. Default can normally be found on the node's datasheet or box. |
 | network_s_key | string | yes | Network Session Key (16 bytes in hex) Used to decypt uplink messages. Default can normally be found on the node's datasheet or box. |
 
+### DoCommand
+
+The sensor node model uses DoCommands to send downlinks to sensors from the gateway.
+
+#### Send a downlink
+
+This command will send a generic downlink payload to the gateway. the string is expected to be a set of bytes in hex.
+
+```json
+{
+  "send_downlink": "<BYTESINHEX>"
+}
+```
+
```

Comment:
I only added documentation for the nodes. I chose not to add documentation for the gateway downlink DoCommand, but we can add it if people want.